### PR TITLE
API-2466: Update confluent_catalog_entity_attributes.md 

### PR DESCRIPTION
**Pull request by linouk23**
### What
This PR updates the TF documentation for the `confluent_catalog_entity_attributes` resource by adding an example of how to set a schema, environment, or Kafka cluster description.

### CODE REVIEW

File: docs/resources/confluent_catalog_entity_attributes.md [hunk 1]
```diff
@@ -33,7 +77,8 @@ resource "confluent_catalog_entity_attributes" "main" {
     secret = "<Schema Registry API Secret for data.confluent_schema_registry_cluster.essentials>"
   }
   
-  entity_name = "lkc-15xq83:topic_0"
+  # example: entity_name = "lkc-15xq83:topic_0"
+  entity_name = "${data.confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}"
```
Comment (cryoshida) [inline docs/resources/confluent_catalog_entity_attributes.md:81]: Nit discrepancy: I see below in description you have this instead
```
${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}.
```

File: docs/resources/confluent_catalog_entity_attributes.md [hunk 2]
```diff
@@ -23,7 +23,51 @@ provider "confluent" {
   cloud_api_secret = var.confluent_cloud_api_secret # optionally use CONFLUENT_CLOUD_API_SECRET env var
 }
 
-resource "confluent_catalog_entity_attributes" "main" {
+resource "confluent_catalog_entity_attributes" "environment" {
```
Comment (cryoshida) [inline docs/resources/confluent_catalog_entity_attributes.md:26]: Also I saw some usage of `confluent_catalog_entity_attributes.main` in the argument reference section, I guess it's `.environmnet` now?

File: docs/resources/confluent_catalog_entity_attributes.md (hunk 1 above)
Comment (linouk23) [reply docs/resources/confluent_catalog_entity_attributes.md:81]: That's a great catch! The value of the `entity_name` depends on the target resource, for example, there's `fka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}` in the _Argument Reference_ block as well:

> - `entity_name` - (Required String) The qualified name of the entity, for example, `${data.confluent_schema_registry_cluster.essentials.id}:.:${confluent_schema.purchase.schema_identifier}`, `${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}`.

File: docs/resources/confluent_catalog_entity_attributes.md (hunk 2 above)
Comment (linouk23) [reply docs/resources/confluent_catalog_entity_attributes.md:26]: I just created APIT-2477 🫡

**Review by cryoshida — APPROVED** [review]

File: docs/resources/confluent_catalog_entity_attributes.md (hunk 1 above)
Comment (linouk23) [reply docs/resources/confluent_catalog_entity_attributes.md:81]: Oh wait, you're right, there's
```
${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}
```
in Argument Reference block but it should have been just 
```
${data.confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}
```
instead.

**Pull request merged**